packages/env/lib/config/read-raw-config-file.js

```diff
@@ -13,26 +14,27 @@ const { ValidationError } = require( './validate-config' );
  * Reads the config JSON from the filesystem and returns it as a JS object
  * compatible with the env package.
  *
- * @param {string} name       The name of the file for error messages.
  * @param {string} configPath The path to the JSON file to read.
  *
  * @return {Object} the raw config data.
  */
-module.exports = async function readRawConfigFile( name, configPath ) {
+module.exports = async function readRawConfigFile( configPath ) {
 	try {
 		return withBackCompat(
```

Comment:
`withBackCompat` might not be needed after your other changes to port parsing 🤔

Reply:
It's not, and, the conversion is covered by other tests.